packages/core/admin/admin/src/pages/MarketplacePage/index.js

```diff
@@ -24,6 +25,16 @@ import useFetchInstalledPlugins from '../../hooks/useFetchInstalledPlugins';
 import useFetchMarketplacePlugins from '../../hooks/useFetchMarketplacePlugins';
 import adminPermissions from '../../permissions';
 
+const matchSearch = (plugins, search) => {
+  return matchSorter(plugins, toLower(search), {
```

Comment:
Looking at the match-sorter docs, I don't think it's necessary to manually change the casing here. If we want to ignore the casing when doing the sort, I think we're supposed to use thresholds instead

Reply:
yeah you are right, I grabbed this as an example from the content manager but actually it seems this additional function really isn't even necessary as well.